c/wakaama

```diff
@@ -1 +1 @@
-Subproject commit edb4a4a50f5f8184b309a79ed55c1294943ce587
+Subproject commit 9fd7bc5a3028f1afdf8908fc974eecb0f6d9484f
```

Comment:
This commit (9fd7bc...) wasn't committed to the wakaama fork.  But this represents some work that needs to happen: CMakeLists.txt in wakaama is expecting a Darwin folder with platform.c and memtrace.c. (creating the folder manually and copying the two files from linux works, as a workaround).